src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -152,7 +154,7 @@ public static void SendTimeout_For_Long_Running_Operation_Throws_TimeoutExceptio
             {
                 IWcfService proxy = factory.CreateChannel();
                 watch.Start();
-                proxy.EchoWithTimeout("Hello");
+                proxy.EchoWithTimeout("Hello", serviceOperationTimeout);
             });
         }
         finally
```

Comment:
Synced up with Matt, short version @iamjasonp yes when checking ranges we need to accommodate for a +/- 15 ms. Apparently that is the interval used when checking the cpu clock to get elapsed time.
@roncain there isn't a bug in NET Native, different implementation there, the sporadic failure is just this clock validation thing.